src/main/java/org/tron/common/runtime/Runtime.java

```diff
@@ -60,6 +72,7 @@
   private String runtimeError;
   private boolean readyToExecute = false;
 
+  CpuProcessor cpuProcessor = null;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Explicitly declare the visibility for "cpuProcessor". ([squid:S2039](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2039))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1188&issues=AWTg8qSOOVxoJwmFDyWT&open=AWTg8qSOOVxoJwmFDyWT)

[ISSUE_KEY]: AWTg8qSOOVxoJwmFDyWT